### ISSUE TITLE
Missing label on table of contents

### ISSUE THREAD

**Issue by imperator99** [govau/dta-gov-au#129]
The table of contents filter is missing a label attribute on the `<form>` tag.

**Comment by imperator99** [comment]
Resolved by https://github.com/govau/dta-website-rebuild/pull/263.
